Dockerfile_Service

```diff
@@ -0,0 +1,25 @@
+FROM nikolaik/python-nodejs:python3.8-nodejs16
+
+RUN apt-get update
+
+# Install serverless
+RUN npm install -g serverless && \
+    npm install -g serverless-offline
+
+WORKDIR /app
+COPY . /app
```

Comment:
Not really an issue, but maybe we could install dependencies first then add the code. This makes it so you don't have to install dependencies again when building locally after a change on the source code.

A good order would be:
- add `package.json` and lock file
- run `npm install`
- add `pyproject.toml` and lock file
- run poetry install
- add source code